### ISSUE TITLE
Can't use client debugger on minified bundle.

### ISSUE THREAD

**Issue by davidkellerman** [nxus/clientjs#24]
The bundled output from clientjs is minified (as one very long line), which means debugging anything in the bundle using the browser debugger is not possible. Certainly, during development at least, un-minified output should be the default.

**Comment by loppear** [comment]
Yeah, I think sourcemaps have to be the answer regardless, open question is probably whether the default choice of source map approach is as useful as it could be (https://webpack.js.org/configuration/devtool/#devtool) - for dev I'm using `cheap-module-eval-source-map` and for production `source-map`.
